docs/superpowers/handovers/2026-04-11-firing-stability-investigation-kickoff.md

```
@@ -0,0 +1,228 @@
+git show main:src/features/input-mapping/mapHandToGameInput.ts
+git show main:src/features/input-mapping/evaluateGunPose.ts
+git show main:src/features/input-mapping/createHandEvidence.ts
+git show main:src/shared/config/gameConfig.ts
+git show main:src/features/hand-tracking/createMediaPipeHandTracker.ts
+```
+
+List more with `git ls-tree main src/features/input-mapping/`.
+
+## Commit history worth skimming
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)